### PR TITLE
 E2E: Implement k8s|oc:log test in vertx-simplest test suites

### PR DESCRIPTION
**Pull request by anandrkskd**
This PR adds end2end test for k8s and oc log in vertex-simplest test suites.

### CODE REVIEW

**Comment by manusa** [comment]
Hi @anandrkskd 
Code looks good, but I'm not sure how you managed to verify this against a real cluster since tests won't compile. (hint: check imports section)

File: it/src/test/java/org/eclipse/jkube/integrationtests/vertx/VertxOcITCase.java [hunk 1]
```diff
@@ -109,6 +109,25 @@ void ocApply() throws Exception {
 
   @Test
   @Order(4)
+  @DisplayName("oc:log, should retrieve log")
+  void ocLog() throws Exception {
+    // Given
+    final Properties properties = new Properties();
```
Comment (rohanKanojia) [inline it/src/test/java/org/eclipse/jkube/integrationtests/vertx/VertxOcITCase.java:115]: Everything looks fine, but whenever you introduce some new class in your code. You need to tell the compiler where it is. That is done using class's fully qualified name(check import statements on top of this file). I think you forgot to tell compiler about it. Just add these lines in import statements(by IDE you can also do it via `Alt + L`:
```
import java.io.ByteArrayOutputStream;
import java.io.PrintStream;
import java.nio.charset.StandardCharsets;
import java.util.Properties;
import org.eclipse.jkube.integrationtests.maven.MavenUtils;

import static org.hamcrest.Matchers.stringContainsInOrder;
```

File: it/src/test/java/org/eclipse/jkube/integrationtests/vertx/VertxK8sITCase.java [hunk 2]
```diff
@@ -122,6 +122,25 @@ void k8sApply() throws Exception {
 
   @Test
   @Order(4)
+  @DisplayName("k8s:log, should retrieve log")
+  void k8sLog() throws Exception {
+    // Given
+    final Properties properties = new Properties();
```
Comment (rohanKanojia) [inline it/src/test/java/org/eclipse/jkube/integrationtests/vertx/VertxK8sITCase.java:128]: same as my previous comment.

File: it/src/test/java/org/eclipse/jkube/integrationtests/vertx/VertxK8sITCase.java [hunk 3]
```diff
@@ -37,13 +43,7 @@
...
 import static org.eclipse.jkube.integrationtests.assertions.YamlAssertion.yaml;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.allOf;
-import static org.hamcrest.Matchers.anEmptyMap;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.hasItems;
-import static org.hamcrest.Matchers.hasProperty;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.*;
```
Comment (manusa) [inline it/src/test/java/org/eclipse/jkube/integrationtests/vertx/VertxK8sITCase.java:46]: We're using Eclipse import style. If you're in IntelliJ Change your settings:
Class count to use import with '*': 999
Names count to use static import with '*': 999
image

**Comment by manusa** [comment]
Please address the issue and squash your commits

**Comment by manusa** [comment]
Closes: eclipse/jkube#133

**Pull request merged**